### ISSUE TITLE
Progress report call - 1st April

### ISSUE THREAD

**Issue by catiatpereira** [ipfs-shipyard/pm-idm#89]
Date: 1st April, 16:00 GMT+0
Notes: https://cryptpad.fr/code/#/2/code/edit/prrhUwo7vfuKuWQKihz1MWEo/
Call: https://zoom.us/j/134161591

**Comment by daviddias** [comment]
Where can we find the recording?

**Comment by satazor** [comment]
@daviddias the call had some technical difficulties: 

- The first 5m of the call were not recorded because I clicked to record, but the recording was not initiated for some reason. I noticed it when we were already 5m in.
- The internet was pretty bad and the call went down several times. This was partly due to being the first call in MOXY’s offices

I still have the recording in my computer but I thought on skipping this one because of the issues we had. I will upload the video and link it in the notes, but I think we should skip uploading this one to IPFS channel maybe.
